tsconfig.json

```diff
@@ -31,6 +31,7 @@
 
     /* Advanced Options */
     "resolveJsonModule": true,
+    "skipLibCheck": true, // @types/webpack and webpack/types.d.ts are not the same thing
```

Comment:
👀

Reply:
Let's use this, as far as I see it's complicated to get rid of `@types/webpack` since it's used in a lot of our dependencies:

```
=> Found "@types/webpack@4.41.26"
info Has been hoisted to "@types/webpack"
info Reasons this module exists
   - "workspace-aggregator-218f88c3-341a-439c-abe9-945d05f720e4" depends on it
   - Hoisted from "_project_#@types#react-dev-utils#@types#webpack"
   - Hoisted from "_project_#@types#react-loadable#@types#webpack"
   - Hoisted from "_project_#@types#webpack-dev-server#@types#webpack"
   - Hoisted from "_project_#@types#webpack-merge#@types#webpack"
   - Hoisted from "_project_#@types#react-dev-utils#@types#html-webpack-plugin#@types#webpack"
```

Reply:
Also for future reference, this does speed up TSC because it doesn't have to go through the extra step of checking typedefs from the `@types` namespace.